README.md

```diff
@@ -5,6 +5,68 @@ https://flutter.dev/
 Flutter in an open Source SDK for creating high-performance mobile apps for IOS and Android.
 The Flutter makes it easier for you to build user interfaces, while reducing the amount of code required to create and update your app.
 
+## Why use Flutter?
+
+- Develop for IOS and Android from a single codebase
+- Do more with less code, with a modern, expressive language and a declarative approach
+- Experiment by changing code and reloading as your app runs (with hot reload)
+- Fix crashes and continue debugging from where the app left off
+- Benefit from a rich set of Material Design and Cupertino (iOS-flavor) widgets built using Flutter’s own framework
+
+## Core Principles
+
+Flutter includes a 2D rendering engine, ready-made widgets, and development tools.</br>
+These components work together to help you design, build, test, and debug apps.</br>
+
+## Widgets
+
+Widgets are the basic building blocks of a Flutter app’s user interface.</br>
+Each widget is an unalterable declaration of part of the user interface.</br>
+Flutter has a consistent, unified object model: the widget.</br>
+
+A widget can define:
+- a structural element (like a button or menu)
+- a stylistic element (like a font or color scheme)
+- an aspect of layout (like padding)
+
+Widgets form a hierarchy based on composition. Each widget nests inside, and inherits properties from, its parent. There is no separate “application” object. Instead, the root widget serves this role.
+
+![Screen Shot 2020-02-06 at 19 01 17](https://user-images.githubusercontent.com/27420533/73969408-4475d280-4913-11ea-8384-99c863321155.png)
+
+Widgets are themselves often composed of many small, single-purpose widgets that combine to produce powerful effects.</br>
```

Comment:
@miguelmartins17 please do not copy-paste directly from the Flutter docs. 
[google search](https://www.google.com/search?q=Widgets+are+themselves+often+composed+of+many+small%2C+single-purpose+widgets+that+combine+to+produce+powerful+effects.&oq=Widgets+are+themselves+often+composed+of+many+small%2C+single-purpose+widgets+that+combine+to+produce+powerful+effects.&aqs=chrome..69i57.209j0j7&sourceid=chrome&ie=UTF-8)
![image](https://user-images.githubusercontent.com/194400/74018037-146b1580-498d-11ea-84ed-7b94d910ce44.png)
What you have added here is ***identical*** to the Flutter Technical overview
https://flutter.dev/docs/resources/technical-overview
![image](https://user-images.githubusercontent.com/194400/74018259-83486e80-498d-11ea-83c6-ad7f5d5a7724.png)
Maybe your teachers did not _care_ about students copy-pasting large blocks of text  but we do.
1. it does not show that you have _learned/understood_ the concepts; it just shows you are being lazy. 
2. You should be learning how to _synthesize_  (_or "summarise" if you prefer_) other people's writing.
3. https://en.wikipedia.org/wiki/Plagiarism

Please re-write this in your own words or simply _link_ to the official docs given that they include images and have been written by professional technical writers.